utils/build-script-impl

```diff
@@ -74,8 +74,8 @@ KNOWN_SETTINGS=(
     swift-build-type            "Debug"          "the CMake build variant for Swift"
     swift-enable-assertions     "1"              "enable assertions in Swift"
     swift-analyze-code-coverage "not-merged"     "Code coverage analysis mode for Swift (false, not-merged, merged). Defaults to false if the argument is not present, and not-merged if the argument is present without a modifier."
-    swift-tools-enable-lto      "0"              "enable LTO compilation of Swift tools. *NOTE* This does not include the swift standard library and runtime."
-    llvm-enable-lto             "0"              "enable LTO compilation of LLVM/Clang."
+    swift-tools-enable-lto      ""               "enable LTO compilation of Swift tools. *NOTE* This does not include the swift standard library and runtime. Must be set to one of thin or full"
+    llvm-enable-lto             ""               "Must be set to one of thin or full"
```

Comment:
Could you put "thin" and "full" in quotes to make it easier to read?  (here and in other help messages)